### ISSUE TITLE
Undefined workspace while trying to generate nx-electron app.

### ISSUE THREAD

**Issue by broken** [bennymeg/nx-electron#171]
**Describe the bug**
Error while trying to generate an nx-electron app.

**To Reproduce**
```
npx create-nx-workspace@13 --preset=core
# Used workspace name 'test'
cd test
npm install -D @nrwl/web
nx g @nrwl/web:app fe
npm install -D nx-electron
nx g nx-electron:app app --frontendProject=fe
# Using . as the directory, but I tried other values as well. I didn't see much info on what is expected here.
```

The error
```
✔ In which directory should the electron application be generated? · .
TypeError: Cannot read properties of undefined (reading 'targets')
    at /Users/dogatech/Development/src/test/node_modules/nx/src/config/workspaces.js:240:27
    at Array.forEach (<anonymous>)
    at toOldFormatOrNull (/Users/dogatech/Development/src/test/node_modules/nx/src/config/workspaces.js:236:37)
    at MapSubscriber.project (/Users/dogatech/Development/src/test/node_modules/nx/src/adapter/ngcli-adapter.js:181:82)
    at MapSubscriber._next (/Users/dogatech/Development/src/test/node_modules/rxjs/internal/operators/map.js:49:35)
    at MapSubscriber.Subscriber.next (/Users/dogatech/Development/src/test/node_modules/rxjs/internal/Subscriber.js:66:18)
    at MapSubscriber._next (/Users/dogatech/Development/src/test/node_modules/rxjs/internal/operators/map.js:55:26)
    at MapSubscriber.Subscriber.next (/Users/dogatech/Development/src/test/node_modules/rxjs/internal/Subscriber.js:66:18)
    at DefaultIfEmptySubscriber._complete (/Users/dogatech/Development/src/test/node_modules/rxjs/internal/operators/defaultIfEmpty.js:45:30)
    at DefaultIfEmptySubscriber.Subscriber.complete (/Users/dogatech/Development/src/test/node_modules/rxjs/internal/Subscriber.js:78:18)
Cannot read properties of undefined (reading 'targets')
```

**Desktop (please complete the following information):**
 - OS: OSX 11.6.5
 - Nx Electron Version: nx-electron@13.2.1
 - Nx Version: nx@13.10.5
 - NPM Version: npm@8.10.0

**Comment by jieey1140** [comment]
I also get the same error as above :(

**Comment by broken** [comment]
I did. I tried no value, `.`, the full path to the packages dir, to a new directory, etc. - all got the same error.

I just tried again, creating a new `v13.10.6` workspace, and still receive the error.

**Comment by broken** [comment]
I just tested this on a Linux machine and got the same error. However, I did some digging online and found [this issue](https://github.com/nrwl/nx/issues/8588). I rolled back my version of nx to 13.4.5 and was able to run the command successfully.

I haven't tested any more up to just seeing it complete, so don't know which version of nx is the first to have this problem, but wanted to post this reply to help any others experiencing the same problem.

**Comment by broken** [comment]
After doing some more testing, it looks like minor version 10 is when the failure started. NX version `13.9.7` (and all before I tried) did not have a problem, but version `13.10.0` is the first that began showing that error.

My guess (with absolutely no experience with nx or nx-electron) is [this pull](https://github.com/nrwl/nx/pull/9710) which removed workspace.json is the culprit. With no workspace.json, the generator for nx-electron cannot find the frontendProject specified.

Yep. I just added a simple workspace.json that would have been there from earlier version and the nx-electron generator using version `13.10.0` of nx now completes without error.

```
{
  "version": 2,
  "projects": {
    "fe": "packages/fe",
    "fe-e2e": "packages/fe-e2e"
  }
}
```

@bennymeg I'm not familiar enough to know how this should be handled - in nx, nx-electron, or documentation, so I'm going to leave how to proceed to you.
